docs/build/build-hrmp-channels.md

```diff
@@ -0,0 +1,111 @@
+
+1. Publication on [Polkasembly](https://polkadot.polkassembly.io/discussions) for discussion and
+   feedback;
+2. Preimage submission on democracy tab (Polkadot JS Apps);
+3. Submission of the proposal as an external motion to Council;
+4. Vote by the community in Democracy module.
+
+:::info
+
+Please note your parachain must make a channel request with Statemint before this proposal enacts,
+in order for the proposal to be enacted successfully.
+
+:::
+
+## Publication on Polkasembly for discussion and feedback
+
+The [discussions section of Polkassembly](https://polkadot.polkassembly.io/discussions) is the best
+place to share the reasoning behind your proposal: make sure to log in with the proposer address (if
+possible) before publishing yours, and if you can do this also make sure the address has an on-chain
+identity.
+
+Your proposal should contain the following sections to be considered complete for the community to
+review and ultimately vote:
+
+1.  A request on what the proposal aims to do (opening an HRMP channel with Statemint);
+2.  The use cases this channel will support for your chain;
+3.  Technical details of the proposal, including proposal parameters and technical details of this
+    call (On Kusama, most proposals were designed as a batchAll calls) :
+
+        - A force transfer from Polkadot treasury to Statemint as deposit to accept and open an HRMP channel with your chain;
+        - Send XCM message to Statemint to execute a transaction with superuser (root) permission.
+
+        Please note that if governance decides to reduce the HRMP channel deposit on Polkadot to 0 DOT, the first transaction should not be necessary (these guidelines will be updated accordingly).
+
+4.  The XCM message to Statemint, which can be decoded on the network;
+5.  The call data to verify on
+    [Polkadot JS Apps Decode](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.polkadot.io#/extrinsics/decode)
+    tab;
+6.  The proposal hash for the external motion submission.
+
+Below is an example of how teams followed this process on Kusama, as a way to:
+
+- Proposal to open HRMP channel between Bifrost and Statemine: the motion can be found
+  [here](https://kusama.polkassembly.io/motion/418).
+
+## Preimage submission on democracy tab (Polkadot JS Apps)
+
+Once the community has given enough feedback (we expect these proposal to be non controversial)
+please submit the image on the discussion post in the
+[Democracy](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.polkadot.io#/democracy) tab on
+Polkadot JS Apps, by using the "Submit Preimage" button:
+
+![submit preimage](../assets/democracy/submit-preimage.png)
+
+On the pop up window, compose the preimage in the discussion post: making sure the proposal hash is
+the same as in the post.
+
+## Submission of the proposal as an external motion to Council
+
+A Council member will need to assist you in order to be able to submit an external motion: they will
```

Comment:
Can be helpful to link to the relevant governance section https://wiki.polkadot.network/docs/learn-governance#council